code/Task_List/app/src/main/java/aleksandr_logunov/task_list/base/BaseFragment.kt

```diff
@@ -14,7 +14,7 @@ abstract class BaseFragment<T : ViewBinding, VM : BaseViewModel?>(@LayoutRes lay
     @Suppress("PropertyName")
     val TAG: String? = javaClass.canonicalName
 
-    var viewModel: VM? = null
+    var viewModel: VM? = null // naming conventions vm
```

Comment:
It's a base class for fragments so what name to choose? Every fragments will use it